next/pages/dane-a-poplatky/index.tsx

```diff
@@ -67,11 +74,17 @@ export const getServerSideProps = amplifyGetServerSideProps<AccountTaxesFeesPage
   { requiresSignIn: true },
 )
 
-const AccountTaxesFeesPage = ({ taxesData, taxAdministrator }: AccountTaxesFeesPageProps) => {
+const AccountTaxesFeesPage = ({
+  taxesData,
+  taxAdministrator,
+  dehydratedState,
+}: AccountTaxesFeesPageProps) => {
   return (
-    <AccountPageLayout>
-      <TaxesFeesSection taxAdministrator={taxAdministrator} taxesData={taxesData} />
-    </AccountPageLayout>
+    <HydrationBoundary state={dehydratedState}>
```

Comment:
(just for fun, no action required) love the search results for this

<img width="850" alt="Screenshot 2024-04-24 at 16 19 10" src="https://github.com/bratislava/konto.bratislava.sk/assets/2327312/98f49fd6-b711-4ad4-8870-d40c9e2510c6">